lfsapi/client.go

```diff
@@ -222,3 +223,7 @@ func newRequestForRetry(req *http.Request, location string) (*http.Request, erro
 	newReq.ContentLength = req.ContentLength
 	return newReq, nil
 }
+
+func init() {
+	UserAgent = config.VersionDesc
```

Comment:
This may be better off in the commands package. That way config and lfsapi are completely decoupled.